vision/object_detection_segmentation/yolov2-coco/README.md

```diff
@@ -0,0 +1,32 @@
+# YOLOv2-COCO
+
+## Description
+This model aims to detect objects in real time. It detects 80 different classes from the [COCO Datasets](http://cocodataset.org/#home). For information on network architecture, see the [author's page](https://pjreddie.com/darknet/yolov2/) and [white paper](https://arxiv.org/pdf/1612.08242.pdf).
+
+## Model
+
```

Comment:
nit: extra whitespace

Reply:
Thank you for your comment. I changed it